unity-renderer/Assets/DCLPlugins/UUIDEventComponentsPlugin/UUIDComponent/PointerEventsController/PointerEventsController.cs

```diff
@@ -51,14 +51,12 @@ public class PointerEventsController
             pointerHoverController.OnPointerHoverStarts += SetHoverCursor;
             pointerHoverController.OnPointerHoverEnds += SetNormalCursor;
 
-            for (int i = 0; i < Enum.GetValues(typeof(WebInterface.ACTION_BUTTON)).Length; i++)
+            for (var id = 0; id < Enum.GetValues(typeof(WebInterface.ACTION_BUTTON)).Length; id++)
```

Comment:
I believe you can also
```suggestion
            foreach (int id in Enum.GetValues(typeof(WebInterface.ACTION_BUTTON)))
```

Reply:
Since it is used only twice - once on Constructor and once in Dispose, perf drop should not be a big deal. I will check how we can improve it (because we also can cache without `ACTION_BUTTON.ANY` and avoid equality check on each iteration)

Reply:
adjustments are in [this commit](https://github.com/decentraland/unity-renderer/pull/4118/commits/7ed67668c911356b2474b62617842085a9cf1062)

Reply:
https://www.reddit.com/r/gamedev/comments/qdfz08/unity_for_vs_foreach_performance/

This used to be true but not anymore, just pick what better suits your needs!